index.html

```diff
@@ -267,17 +272,23 @@ <h2 class="heading">
             </ul>
         </div>
     </section>
+    <!--Contact form-->
     <section class="contact">
         <div class="contact-heading">
-            <div>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+            <div>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </div>
         </div>
         <div class="form">
-            <input class="form-fill" type="text" name="name" placeholder="Full name">
-            <input class="form-fill" type="email" name="email" placeholder="Email address">
-            <textarea name="mail-content" id="" cols="30" rows="5" placeholder="Enter text here"></textarea>
-            <input class="submit" type="submit" value="Get in touch">
+            <form action="https://formspree.io/f/xknarzjq" method="post">
+                <input class="form-fill" type="text" name="name"  pattern="^[a-zA-Z]{3,30}$" placeholder="Full name" required>
+                <input class="form-fill" type="email" name="email" pattern="^([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$" placeholder="Email address" required>
+                <textarea name="mail-content" id="text" pattern="^[a-zA-Z0-9]{1,500}$" cols="30" rows="5"
+                    placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] AS mentioned above please use the maxlength attribute to keep the messages up to 500 characters